packages/checkout/sdk/src/widgets/definitions/configurations/widget.ts

```diff
@@ -8,6 +8,8 @@ export type WidgetConfiguration = {
   /** The theme of the Checkout Widget (default: "DARK") */
   theme?: WidgetTheme;
   language?: WidgetLanguage;
+  /** Enable/disable Wallet Connect */
+  walletConnect?: boolean;
```

Comment:
```suggestion
  walletConnect?: {
     enable: boolean
  },
```

Reply:
Sorry @mimi-imtbl , this is what I meant by remove the arrays for now. We want to be able to add them in later without it being a breaking change to the config object